server/models/AccountingCategory.ts

```diff
@@ -133,8 +132,11 @@ AccountingCategory.init(
       type: DataTypes.ENUM(...Object.keys(ExpenseTypes)),
```

Comment:
Not sure if that's related to our issue, but I realize this should be:

```suggestion
      type: DataTypes.ARRAY(DataTypes.ENUM(...Object.keys(ExpenseTypes))),
```

Reply:
Awesome!